docker/orb.yaml

```diff
@@ -362,6 +362,33 @@ commands:
               docker push "<<parameters.registry>>/<<parameters.image>>:${CIRCLE_TAG}"
             fi
 
+  push-with-custom-tag:
```

Comment:
Would you mind linking to or explaining why this is different in use-case than `push-with-tag`? So far, we've found it valuable to use `$CIRCLE_TAG` (which is the name of the git tag which triggered this build) to ensure that our image tags and git tags are kept in sync.

Reply:
sure - here is the link to the PR were I want to use this functionality and derive from using `$CIRCLE_TAG` https://github.com/talkiq/offline/pull/1050/files. The idea is for the multi-platform gradle base image builder to have the the target gradle version as the tag that's being pushed. I defined the gradle version as configuration for the `run-build-gradle-mutlipatform-base-image` workflow.